core/src/main/scala/org/broadinstitute/dsde/rawls/monitor/workspace/runners/BPMBillingProjectDeleteRunner.scala

```diff
@@ -92,13 +93,21 @@ class BPMBillingProjectDeleteRunner(
   ): Future[JobStatus] =
     Try(workspaceManagerDAO.getDeleteLandingZoneResult(jobId.toString, lzId, ctx)) match {
       case Failure(e: ApiException) =>
-        val msg =
-          s"API call to get Landing Zone deletion operation failed with status code ${e.getCode}: ${e.getMessage}"
-        e.getCode match {
-          case code if code >= 400 && code < 500 =>
-            billingRepository.updateCreationStatus(projectName, DeletionFailed, Some(msg)).map(_ => Complete)
-          case _ =>
-            billingRepository.updateCreationStatus(projectName, Deleting, Some(msg)).map(_ => Incomplete)
+        if (e.getCode == StatusCodes.Forbidden.intValue) {
```

Comment:
Instead of nesting the if statement, you could just add this as first case:
```
      case Failure(e: ApiException) if e.getCode == StatusCodes.Forbidden.intValue =>
        logger.info(
          s"LZ deletion result status = ${e.getCode} for LZ ID ${lzId}, LZ record is gone. Proceeding with rawls billing project deletion"
        )
        billingProjectDeletion.finalizeDelete(projectName, ctx).map(_ => Complete)
```

Reply:
Good idea, implemented.